test/bridge-executor/arbitrum.integration.test.ts

```diff
@@ -24,11 +24,35 @@ scenario("Arbitrum :: Bridge Executor integration test", ctx)
     await ctx.l2.provider.send("evm_revert", [ctx.snapshot.l2]);
   })
 
+  .step("Check initial setup", async (ctx) => {
+    const {
+      l2: { bridgeExecutor, l2ERC20TokenGateway },
+    } = ctx;
+    const actionsSetCount = await bridgeExecutor.getActionsSetCount();
+
+    assert.equal(actionsSetCount.toNumber(), 0);
```

Comment:
This check works only for the freshly deployed contract. If we run such a test on existed contract, it might already have action sets (for example, our instance of executor). It's better to put all these checks in the "Activate Bridging" step and retrieve `actionsSetCount` before tx and after, and compare them.